### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by meatballs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,23 +20,18 @@ A Python tool to assist the task of scheduling a conference which:
...
-# Built-In Optimisations
+# Optimisation
 * The sum of 'potential disappointments' should be minimised where 'potential disappointments' is defined as the excess of demand over room capacity for every scheduled event
-* The number of slot/talk combinations without a person assigned to the 'Session chair' role should be minimised
-* The number of occasions where the session chair changes within a session for any room should be minimised
 
-# Ad-Hoc Constraints and Optimisations
-Some examples of situations which have arisen at previous conferences and should be handled by defining ad-hoc constraints or optimisations:
+# Examples
+Some examples of situations which have arisen at previous conferences and chould be handled by the unavailability, clashing and tagging constraints:
```
Comment (alexwlchan) [inline README.md]: /s/chould/could

Comment (drvinceknight) [inline README.md]: Doh! Sorry I missed this and merged it, although I think `could` is correct here. Not sure: I can hardly speak anyway.

Comment (drvinceknight) [inline README.md]: @meatballs just in case you somehow miss @alexwlchan's comment because of my fast merge. :arrow_up:

Comment (drvinceknight) [inline README.md]: https://github.com/PyconUK/ConferenceScheduler/pull/40/files

Comment (meatballs) [inline README.md]: pushed the fix to master

**Pull request merged**